src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.dao;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionDao extends GenericDao<MovieSession> {
+    List<MovieSession> get(Long movieId, LocalDate date);
```

Comment:
`get` it's not informative method name, lets rename it to `findAvailableSessions` or smth like this

Reply:
Fixed.